stl/src/awint.h

```diff
@@ -220,10 +220,24 @@ BOOLEAN __cdecl __crtTryAcquireSRWLockExclusive(__inout PSRWLOCK);
 #define __crtGetSystemTimePreciseAsFileTime(lpSystemTimeAsFileTime) \
     GetSystemTimePreciseAsFileTime(lpSystemTimeAsFileTime)
 
+
+#define __crtWaitOnAddress(Address, CompareAddress, AddressSize, dwMilliseconds) \
+    WaitOnAddress(Address, CompareAddress, AddressSize, dwMilliseconds)
+
+#define __crtWakeByAddressSingle(Address) WakeByAddressSingle(Address)
+#define __crtWakeByAddressAll(Address)    WakeByAddressAll(Address)

```

Comment:
Should these be variadic, e.g.:
```c++
#define __crtWaitOnAddress(...) WaitOnAddress(__VA_ARGS__)
```
or object-like instead of function-like macros to avoid the preprocessor's issues with commas outside of parentheses? Should they have names in the macro reserved namespace, so that callers always use e.g. `_CRT_WAIT_ON_ADDRESS` which is conditionally defined to either `WaitOnAddress` or `__crt_WaitOnAddress`? (Defining macros with non-macro names is not a great practice.)

Reply:
I copied style of existing definitions, like `__crtGetSystemTimePreciseAsFileTime` on line 220 just above.
Anyway, as these macros are used only internally, I expect no problems.
But definitely can be improved by using inline function instead of macro, should I do that?

Reply:
Actually, inline function will collide with the one in .cpp. So will fix macro as you propose.

Reply:
obsoleted as reverted `awint.h` changes